app/models/lesson_schedule.rb

```diff
@@ -0,0 +1,15 @@
+class LessonSchedule < ApplicationRecord
+  belongs_to :course
```

Comment:
Add this line:
  self.inheritance_column = :_type_disabled # So that we can use the :type column